build/AzurePipelinesTemplates/WindowsAppSDK-PackNuget-Steps.yml

```diff
@@ -0,0 +1,166 @@
+parameters:
+- name: "SignOutput"
+  type: boolean
+  default: False
+
+steps:
+- task: DownloadPipelineArtifact@2
+  displayName: 'Download Foundation x64'
+  inputs:
+    artifactName: "drop_TransportPackage_BuildBinaries_release_x64"
+    targetPath: '$(Build.SourcesDirectory)\BuildOutput'
+- task: DownloadPipelineArtifact@2
+  displayName: 'Download Foundation x86'
+  inputs:
+    artifactName: "drop_TransportPackage_BuildBinaries_release_x86"
+    targetPath: '$(Build.SourcesDirectory)\BuildOutput'
+- task: DownloadPipelineArtifact@2
+  displayName: 'Download Foundation arm64'
+  inputs:
+    artifactName: "drop_TransportPackage_BuildBinaries_release_arm64"
+    targetPath: '$(Build.SourcesDirectory)\BuildOutput'
+
+- task: DownloadPipelineArtifact@2
+  displayName: 'Download Foundation arm64'
+  inputs:
+    artifactName: "drop_TransportPackage_BuildBinaries_release_anycpu"
+    targetPath: '$(Build.SourcesDirectory)\BuildOutput'
+
+- task: DownloadPipelineArtifact@2
+  displayName: 'Download Foundation x64'
```

Comment:
Download Foundation MRT x64? To distinguish this from the case above?
Similar for the following 2 downloads.